src/Framework/BuildCop/BuildCopEventArgs.cs

```diff
@@ -20,6 +20,8 @@ public sealed class BuildCopTracingEventArgs(Dictionary<string, TimeSpan> tracin
     internal BuildCopTracingEventArgs() : this(new Dictionary<string, TimeSpan>())
     { }
 
+    internal bool isInfraTracing = false;
```

Comment:
Do we need to distinguish the infra and analyzers/checks tracing stats?

Both - infrastructure and checks - will run in main and worker nodes - so we'll need to transfer and merge both.

We can probably even just push them into the single dictionary (they can differ by prefix of the key e.g.) - that way we can have single message for transfer and single merging code